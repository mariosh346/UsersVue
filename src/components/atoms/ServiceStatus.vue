<template>
  <q-td>
    <q-icon
      :name="active ? 'check' : 'close'"
      :color="active ? 'positive' : 'negative'"
    />
  </q-td>
</template>

<script setup lang="ts">
import { defineProps } from 'vue';

const props = defineProps({
  props: Object,
  active: Boolean
});
</script>
