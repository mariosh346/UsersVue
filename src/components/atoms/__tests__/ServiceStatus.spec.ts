import { describe, it, expect } from 'vitest';
import { shallowMount } from '@vue/test-utils';
import ServiceStatus from "../ServiceStatus.vue";

describe('ServiceStatus', () => {
  const createWrapper = (props: Record<string, unknown>) =>
    shallowMount(ServiceStatus, { props });

  it('matches snapshot for active false', () => {
    const wrapper = createWrapper({ active: false });
    expect(wrapper.html()).toMatchSnapshot();
  });
});
