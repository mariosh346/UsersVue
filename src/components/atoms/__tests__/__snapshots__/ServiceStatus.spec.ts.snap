// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ServiceStatus > matches snapshot for active false 1`] = `"<q-td-stub autowidth="false" nohover="false"></q-td-stub>"`;
